common/src/View/MapFrame.cpp

```diff
@@ -810,6 +811,20 @@ namespace TrenchBroom {
             }
         }
 
+        bool MapFrame::revertDocument() {
+            if (!m_document->persistent()) {
```

Comment:
Should this not be the same condition that disables the action itself?

Reply:
Yeah. The problem is, `persistent()` does IO, which we shouldn't do in the action enabling/disabling.
IMO it should just check if the path is absolute (a "newmap.map" that's never been saved vs a file on disk) and not bother checking if the file has been deleted since opening..  I'll file another issue about fixing that.